pkg/inits/config.go

```diff
@@ -27,7 +27,7 @@ func GetConfig() *entities.Config {
 	once.Do(func() {
 		viper.SetConfigName("config")
 		viper.SetConfigType("yaml")
-		viper.AddConfigPath("../../")
+		viper.AddConfigPath("/root")
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/aakashgusain1990/productivite/pull/12#issuecomment-2424992384>#7bce26</a><span> - 10/20/2024, 07:48 pm</span>
<br>

<div><small>🔴</small> High importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> Using '/root' directory for configuration files in a Docker container is not a best practice and may pose security risks.
</div>
<div id="fix">
<b>Fix:</b> Use a non-root user and a more secure location for storing configuration files in the Docker container. Consider using environment variables or a dedicated config directory.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -30 +30 @@
 -		viper.AddConfigPath("/root")
 +		viper.AddConfigPath("/app/config")
```
</div>
</details>
</div>
<br>


</div>

Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged